src/components/GradebookFilters/CourseGradeFilter/index.jsx

```diff
@@ -48,28 +50,32 @@ export class CourseGradeFilter extends React.Component {
     } = this.props;
     return (
       <>
-        <div className="grade-filter-inputs">
```

Comment:
Similar to another comment...

[curious] Is there another implementation approach here besides using `Row` and `Col`? It's generally a code smell to need to mess with the horizontal padding on columns in the grid system.

Would treating a container `div` as `d-flex` be another reasonable solution here?